config/locales/hyrax.en.yml

```diff
@@ -257,13 +257,21 @@ en:
       workflow_roles:
         header: Workflow Roles
         index:
+          current_group_roles: Current Group Roles
           current_roles: Current Roles
+          current_user_roles: Current User Roles
           delete:
             confirm: Are you sure?
           header:
+            display_name: Display Name
```

Comment:
remove line 266-269 for this and all other languages

Reply:
Removed lines 266-269 for all languages.